### DISCUSSION TITLE
npm install error

### DISCUSSION THREAD

**Discussion by Alexan89** [microsoft/hack-together-teams#13] [Q&A] [answered]
Hello!
When use the tab template with react/fluentUI i get the following error when running "npm install"

![image](https://github.com/microsoft/hack-together-teams/assets/31478701/1c84cb5c-0774-4490-bca3-8dc2fa6d7165)

I have node version 18.16.0 and i have tried 16 and still get the same error.

If i remove
        "@microsoft/mgt-react": "^2.11.0",
From package.json i can run npm install

**Comment by waldekmastykarz** [comment]
@therealjohn @sebastienlevert @gavinbarron any ideas what could be wrong?

**Reply by gavinbarron** [reply]
This is a problem that has come up since a release of mgt yesterday. We're looking into a fix. 

As a hack workaround you could try editing the package.json for mgt-react to remove the peer dependency on React @17

**Comment by sebastienlevert** [comment]
Hello @Alexan89! We are aware and are working on this and will be providing an overall fix. 

An option was provided here also : https://github.com/OfficeDev/TeamsFx/issues/8908#issuecomment-1573799575

**Reply by sebastienlevert** [reply]
The fix will be available today. I'll keep you posted when it's available on npm to re-run your project.

**Reply by gavinbarron** [reply, marked as answer]
@Alexan89 a fix has been shipped in v2.11.1 of `@microsoft/mgt-react` you should be able to use the default template without error.

**Reply by Alexan89** [reply]
Thnx for the fast response and fix! :)
